README.md

```diff
@@ -7,20 +7,58 @@ Tracks for iOS is a client library used to help track events inside of an applic
 
 ## Installation
 
-You can install the Tracks component in your app via [CocoaPods](http://cocoapods.org):
+
+You can install the Tracks component in your app via Swift Package Manager:
+
+```swift
+.package(url: "https://github.com/Automattic/Automattic-Tracks-iOS", from: "0.10.0")
```

Comment:
I'm making the assumption that when this is merged in, we'll make a "0.10.0" tag